_includes/landingPage.html

```diff
@@ -51,16 +51,14 @@ <h2>Test Anything</h2>
         <div class="span4">
           <h2>Test Containerized</h2>
           <p> Have automated test executions that run locally exactly as they run in the cloud.
-              Test execution is performed within <em>Docker containers</em> that are isolated and complete,
-              such that the identical container is used locally as it is used in the cloud.
-            <br>
+          <br>
             <div class="get-button">
                 <a class="web-button-grey" href="/te_markdown/testexec/">Learn More ...</a> 
             </div>
           </p>
         </div>
         <div class="span4">
-          <h2>Test in the Web</h2>
+          <h2>Test in <em>Your</em> Browser</h2>
```

Comment:
I'm not sure if we want the emphasis on "Your" in this case. On the one hand, its consistent with the other two headings, on the other hand, I think in this case it is misleading, because the point we want to get across is that it is a web-based application, i.e. running in *a* browser. There is nothing special about the reader's own browser, in fact, the point is that Test-Editor-Web runs in *any* browser.

So, maybe no emphasis in this case…? :man_shrugging: 
```suggestion
          <h2>Test in Your Browser</h2>
```